app/experimenter/nimbus-ui/src/components/TableHighlights/index.tsx

```diff
@@ -50,13 +50,17 @@ const TableHighlights = ({
         {Object.keys(results).map((branch) => {
           const userCountMetric =
             results[branch]["branch_data"][METRIC.USER_COUNT];
+          const participantCount =
+            userCountMetric[BRANCH_COMPARISON.ABSOLUTE].length > 0
+              ? userCountMetric[BRANCH_COMPARISON.ABSOLUTE][0]["point"]
+              : null;
```

Comment:
Same comment as below, but not sure if using a temp variable for the `userCountMetric[BRANCH_COMPARISON.ABSOLUTE]` lookup might DRY the code out a bit and shorten the code significantly.

```js
const metric = userCountMetric[BRANCH_COMPARISON.ABSOLUTE];
const participantCount = metric.length ? metric[0].point : null;
```

Not sure if there is a cleverer way that avoids the inline if, maybe `const participantCount = metric[0]?.point;` but maybe that's too much magic happening and I'm just wildly guessing without actually testing that code.